### PR TITLE
Enlarge volume size for increasing backup time

### PR DESCRIPTION
**Pull request by derekbit**
Fix some failed test cases by increasing volume size after introducing fast lz4 compression method.

longhorn/longhorn#5189

### CODE REVIEW

File: manager/integration/tests/test_basic.py [hunk 1]
```diff
@@ -4874,11 +4874,12 @@ def backup_failed_cleanup(client, core_api, volume_name, volume_size,  # NOQA
...
-        'pos': 0,
-        'content': common.generate_random_data(volume_size),
-    }
-    write_volume_data(vol, data)
+    volume_endpoint = get_volume_endpoint(vol)
+
+    # Take snapshots4 without overlapping
+    data_size = volume_size/Mi
+    write_volume_dev_random_mb_data(
+        volume_endpoint, 0, data_size)
```
Comment (shuo-wu) [inline manager/integration/tests/test_basic.py:4882]: NIT: any difference here...?

**Review by shuo-wu — APPROVED** [review]

File: manager/integration/tests/test_basic.py (hunk 1 above)
Comment (derekbit) [reply manager/integration/tests/test_basic.py:4882]: `generate_random_data` is slow when creating a big file, so use `/dev/urandome` instead.

Size is updated here.
```
backup_name = backup_failed_cleanup(client, core_api, volume_name, 1024*Mi)
```

File: manager/integration/tests/test_recurring_job.py [hunk 2]
```diff
@@ -854,7 +854,7 @@ def test_recurring_jobs_when_volume_detached_unexpectedly(set_random_backupstore
     deployment = make_deployment_with_pvc(deployment_name, pvc_name)
     create_and_wait_deployment(apps_api, deployment)
 
-    size_mb = 500
+    size_mb = 1000
```
Comment (joshimoo) [inline manager/integration/tests/test_recurring_job.py:857]: NIT: this should probably be 1024 if you want to write half the volume

**Comment by joshimoo** [comment]
@longhorn/qa do we have enough resources on the test runners/s3 buckets, to increase the volume size to 2GB for this test?

**Comment by khushboo-rancher** [comment]
@joshimoo Yes, 2 GB should be fine. We use minio and NFS backupstore of 4Gi to test backup related tests https://github.com/longhorn/longhorn-tests/tree/master/manager/integration/deploy/backupstores

**Comment by yangchiu** [comment]
@Mergifyio backport v1.4.x

**Comment by roger-ryao** [comment]
@Mergifyio backport v1.3.x

**Pull request merged**